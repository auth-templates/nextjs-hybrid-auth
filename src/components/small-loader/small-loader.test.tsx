import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';
import SmallLoader from './small-loader';

describe('SmallLoader', () => {
    it('renders correctly', () => {
        const tree = renderer.create(<SmallLoader />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});