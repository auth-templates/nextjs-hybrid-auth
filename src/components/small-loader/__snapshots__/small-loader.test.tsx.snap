// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SmallLoader renders correctly 1`] = `
<div
  className="ldsRing"
  data-testid="small-lds-ring"
>
  <div />
  <div />
</div>
`;
